warehouse/templates/pages/help.html

```diff
@@ -149,6 +151,12 @@ <h3 id="APIs">{{ APIs() }}</h3>
         <h3 id="mirroring">{{ mirroring() }}</h3>
         <p>
           If you need to run your own mirror of PyPI, the <a href="https://pypi.org/project/bandersnatch/">bandersnatch project</a> is the recommended solution. Note that the storage requirements for a PyPI mirror would exceed 1 terabyte—and growing!
+        </p>
+
+        <h3 id="notifications">{{ notifications() }}</h3>
+        <p>
+          While PyPI itself does not offer a way to get notified when new releases are uploaded, there are several third-party services that offer comprehensive monitoring and notifications for package releases and vulnerabilities such as <a href="https://pyup.io" target="_blank">pyup.io</a> and <a href="https://libraries.io/" target="_blank">libraries.io</a>.
```

Comment:
When using `target="_blank_"` always set `rel="noopener"`

Reply:
Good catch, done in 9dfff54a.